.github/workflows/run_release_code_and_api.yml

```diff
@@ -88,7 +92,7 @@ jobs:
         with:
           TARGET_ENVIRONMENT: int
           RELEASE_TAG: ${{ inputs.VERSION_NUMBER }}
-          CONFLUENCE_PAGE_ID: "768067994"
+          CONFLUENCE_PAGE_ID: "789747650"
```

Comment:
This shouldn't have been changed, 789747642 is the PSU page.

Reply:
The correct number has been restored.